src/ralph/models/edx/server.py

```diff
@@ -14,6 +16,12 @@ class ServerEventField(BaseModelWithConfig):
     POST: dict
 
 
+@ModelSelector.add(
+    [
+        Rule(ModelField("event_source"), "server"),
+        Rule(ModelField("event_type"), ModelField("context>path")),
```

Comment:
Using dots notation would feel more natural for python developers?

```suggestion
        Rule(ModelField("event_type"), ModelField("context.path")),
```

Reply:
Good point! Yes, indeed, and with the edX events this should work too, but xAPI events use sometimes URI's as keys (containing the `.`)

Some propositions:

- use `.` for edX and `>` for xAPI
- keep `>` for both
- put the keys in a list `[key1, key2]`

WDYT ? Are there other/better solutions ?

Reply:
Oh crap! You are right, it's preferable to use the `>` path notation.